kbfsfuse/kbfsfuse.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+_term() {
+    echo "Caught TERM signal"
+    kill -TERM "$SERVICE"
+    kill -TERM "$KBFS"
+    exit 0
+}
+trap _term SIGTERM
+
+keybase service &
+SERVICE=$!
+kbfsfuse -debug -mdserver mdserver.kbfs.local:8125 -bserver bserver.kbfs.local:8225 -enable-sharing-before-signup /keybase &
```

Comment:
I think the mdserver and bserver strings should be env vars, right?  No reason to force others into the naming/port conventions used in our closed-source code.